front/app/components/VoteControl/VotingDisabled.test.tsx

```diff
@@ -8,7 +8,7 @@ const defaultProps = {
   projectId: 'projectId',
 };
 
-const mockProject = getProject(
+let mockProject: {} | null = getProject(
```

Comment:
Couldn't have IProjectData here instead of {}

Reply:
This would be against our new linter rule as we don't allow { } as type anymore. Can you change it to `Record<string, unknown>` or worst case `Record<string, any>` if it's hard to come up with a more specific type?

Reply:
Was able to do it with IProjectData, although it's through type casting... Without, I get a weird error where it says `'project'` is not compatible with type `string` (for the type attribute of the mock project data) 🤔 Anyway, going to merge like this.

https://github.com/CitizenLabDotCo/citizenlab/pull/597/commits/687efe4b2cdb8f6cb8114efe759aa742004aabf1